gammapy/obs/tests/test_observation.py

```diff
@@ -9,3 +13,97 @@
 
 def test_ObservationTable():
     ObservationTable()
+
+def test_filter_observations():
+    # create random observation table
+    observatory_name='HESS'
```

Comment:
Please put `observatory_name='HESS'` and `n_obs=10` as default keyword arguments for `make_test_observation_table`.
So that if the user doesn't look up and give any parameters, they still get a nice small table to play with.